sdk/sdk/src/test/java/org/apache/carbondata/sdk/file/CarbonIUDTest.java

```diff
@@ -72,6 +72,43 @@ public void testDelete() throws Exception {
     FileUtils.deleteDirectory(new File(path));
   }
 
+  @Test
+  public void testUpdateOnDateType() throws Exception {
```

Comment:
I think this testcase is for update on date type but here you have updated on int type. Please check once.

Reply:
Testcase was added to check conversion of dateType columns to integer while reading. As date column is present in schema. so no need to perform update on date column.